src/Maker/MakeEntity.php

```diff
@@ -56,7 +57,7 @@ public function __construct(FileManager $fileManager, DoctrineHelper $doctrineHe
 
         if (null === $generator) {
             @trigger_error(sprintf('Passing a "%s" instance as 4th argument is mandatory since version 1.5.', Generator::class), E_USER_DEPRECATED);
-            $this->generator = new Generator($fileManager, 'App\\');
+            $this->generator = new Generator($fileManager, new NamespacesHelper());
```

Comment:
Do we need to also pass `root_namespace` set to `App\\`?

Reply:
If not given the `root_namespace` will default to `App\\`, we can probably explicitly set it to avoid confusion.

Any thoughts?